core/src/main/java/com/linecorp/armeria/server/ParameterizedPathMapping.java

```diff
@@ -86,7 +90,7 @@
      * Create a {@link ParameterizedPathMapping} instance from given {@code pathPattern}.
      *
      * @param pathPattern the {@link String} that contains path params.
-     *             e.g. {@code /users/{name}} or {@code /users/:name}
+     *                    e.g. {@code /users/{name}}, {@code /users/:name} or {@code /users/{*name}}
```

Comment:
Q: Should we also support `:*name` as well?

Reply:
Thanks for the suggestion. Sounds good.
[path-to-regexp](https://github.com/pillarjs/path-to-regexp) which javascript library, [Vue Router](https://router.vuejs.org/guide/essentials/dynamic-matching.html) is using supports `/:name*` to denote a zero or more parameter matches.

So we'd better use `/:name*` instead of `:*name` if there is nothing else that supports `:*name` ? 
What do you think, @trustin  ?

Reply:
I guess Trustin suggested symmetrical patterns.
- `{name}` vs `:name` 
- `{*name}` vs `:*name`

Reply:
I see. Let me try to fix it more for `:*name` 😄

Reply:
Thanks! I didn't know about `path-to-regexp`. I guess in our case it's probably better being consistent (have `*` in the beginning). Please let me know if you think we should put `*` after the param name.